services/manifest-repo-export-service/pkg/repository/transformer.go

```diff
@@ -364,54 +364,20 @@ func (c *DeployApplicationVersion) Transform(
 		return "", fmt.Errorf("release of app %s with version %v not found", c.Application, c.Version)
 	}
 	var manifestContent = []byte(version.Manifests.Manifests[c.Environment])
-	if c.LockBehaviour != api.LockBehavior_IGNORE {
-		// Check that the environment is not locked
-		var (
-			appLocks, envLocks, teamLocks map[string]Lock
-			err                           error
-		)
-		envLocks, err = state.GetEnvironmentLocksFromDB(ctx, transaction, c.Environment)
-		if err != nil {
-			return "", err
-		}
-		appLocks, err = state.GetEnvironmentApplicationLocksFromDB(ctx, transaction, c.Environment, c.Application)
-		if err != nil {
-			return "", err
-		}
 
-		app, err := state.DBHandler.DBSelectExistingApp(ctx, transaction, c.Application)
-		if err != nil {
-			return "", err
-		}
-		var teamName = ""
-		if app != nil && app.Metadata.Team != "" {
-			teamName = app.Metadata.Team
-		}
+	/*
+		We don't check for locks because either the Deployment:
```

Comment:
I think there's no need for this comment in this part of code. It could just be in the PR description.